flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/operators/transform/RowFilter.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.runtime.operators.transform;
+
+import com.ververica.cdc.common.data.RecordData;
+import com.ververica.cdc.common.data.binary.BinaryRecordData;
+import com.ververica.cdc.common.schema.Column;
+import com.ververica.cdc.common.utils.StringUtils;
+import com.ververica.cdc.runtime.parser.JaninoParser;
+import com.ververica.cdc.runtime.parser.TransformParser;
+import com.ververica.cdc.runtime.typeutils.DataTypeConverter;
+import org.codehaus.janino.ExpressionEvaluator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/** The RowFilter applies to describe the row filter of filtering tables. */
+public class RowFilter {
+    private static final Logger LOG = LoggerFactory.getLogger(RowFilter.class);
+    private final String expression;
+    private final String scriptExpression;
+    private final List<String> columnNames;
+    private ExpressionEvaluator expressionEvaluator;
+
+    public RowFilter(String expression, String scriptExpression, List<String> columnNames) {
+        this.expression = expression;
+        this.scriptExpression = scriptExpression;
+        this.columnNames = columnNames;
+    }
+
+    public static RowFilter of(
+            String expression, String scriptExpression, List<String> columnNames) {
+        return new RowFilter(expression, scriptExpression, columnNames);
+    }
+
+    public static Optional<RowFilter> generateRowFilter(String filterExpression) {
+        if (StringUtils.isNullOrWhitespaceOnly(filterExpression)) {
+            return Optional.empty();
+        }
+        List<String> columnNames = TransformParser.parseFilterColumnNameList(filterExpression);
+        String scriptExpression =
+                TransformParser.translateFilterExpressionToJaninoExpression(filterExpression);
+        return Optional.of(of(filterExpression, scriptExpression, columnNames));
+    }
+
+    public boolean run(BinaryRecordData after, TableInfo tableInfo) {
```

Comment:
run -> process

Reply:
Could you also change applyXXEvent to processXXEvent in this PR to keep same style with Flink code base ?